### ISSUE TITLE
Missing Com.Microsoft.Applicationinsights.Library.ApplicationInsights

### ISSUE THREAD

**Issue by mike-rowley** [Microsoft/ApplicationInsights-Xamarin#15]
I am trying to rebuild the solution to fix the TrackMetric properties issue I ran across however I am missing the reference Com.Microsoft.Applicationinsights.Library.ApplicationInsights and cannot for the life of me figure out how to get it added.  

Any assistance would be appreciated.
Thanks
M

**Comment by ghost** [comment]
Have you already added a reference to the lokal NuGet package as described in [4.1 Setup](https://github.com/Microsoft/ApplicationInsights-Xamarin#4)?

**Comment by mike-rowley** [comment]
I am able to get the NuGet package installed and can use it no problem.  The issue is trying recompiling the SDK that package is based on to fix the bug I found in TrackMetric (there is a pull request for the issue I found)

I just reopened the solution to find the exact error and project file name it was occurring in and now it is finding the missing objects in AI.XamarinSDK.AndroidBindings\obj\Debug\generated\src

I did everything I could think of including rebooting, solution clean and rebuild etc.  Either way its working now, I appreciate you checking in on this.

Thanks
M
